### PR TITLE
Update whats-new-in-defender-for-office-365.md

### PR DESCRIPTION
**Pull request by VipulMSFT**
Updating what's new to accommodate for new changes coming in December/January

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -47,6 +47,11 @@ For more information on what's new with other Microsoft Defender security produc
 - [What's new in Microsoft Cloud App Security](/cloud-app-security/release-notes)
 
 
+## December/January 2021
+
+- [Updated Hunting and Investigation Experiences for Microsoft Defender for Office 365](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/updated-hunting-and-investigation-experiences-for-microsoft/ba-p/3002015): Introducing the email summary panel for MDO experiences along with experience updates for Threat Explorer and Real-time detections  .
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- [Updated Hunting and Investigation Experiences for Microsoft Defender for Office 365](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/updated-hunting-and-investigation-experiences-for-microsoft/ba-p/3002015): Introducing the email summary panel for MDO experiences along with experience updates for Threat Explorer and Real-time detections.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @VipulMSFT, kindly commit the minor suggested change. Many thanks.

**Comment by VipulMSFT** [comment]
> Copy review - @VipulMSFT, kindly commit the minor suggested change. Many thanks.

@JohanFreelancer9 - committed the change.  Thank you for the suggestion

**Review by denisebmsft — APPROVED** [review]
Thank you @VipulMSFT

**Pull request merged**